prow/io/providers/providers.go

```diff
@@ -67,8 +67,12 @@ func GetBucket(ctx context.Context, s3Credentials []byte, path string) (*blob.Bu
 	if storageProvider == providerS3 && len(s3Credentials) > 0 {
```

Comment:
Might be the better solution to implement your original idea, configuring s3 via config file and only configure access key and secret key when set in the json file.

Reply:
Agree! Simple and doesn't involve multiple files :). Should I proceed?

Reply:
I think so. @alvaroaleman wdyt?

Reply:
yeah, sounds much more elegant